lib/msal-browser/src/controllers/NestedAppAuthController.ts

```diff
@@ -159,7 +159,11 @@ export class NestedAppAuthController implements IController {
      * Specific implementation of initialize function for NestedAppAuthController
      * @returns
      */
-    async initialize(request?: InitializeApplicationRequest): Promise<void> {
+    async initialize(
+        request?: InitializeApplicationRequest,
+        // eslint-disable-next-line @typescript-eslint/no-unused-vars
+        isBroker?: boolean
```

Comment:
Prefixing with _ should tell the linter it's expected for this to be unused
```suggestion
        _isBroker?: boolean
```

Reply:
It isn't working unfortunately